reference/promise-types/packages.markdown

```diff
@@ -526,3 +526,34 @@ Manage packages using Slackware's [slackpkg](https://slackpkg.org).
 **History:**
 
 * Added in CFEngine 3.12.0
+
+### msiexec
+
+Manage MSI packages using MSI installer on Windows.
+
+Due to lack of central software repository on supported versions of Windows,
+neither installation from repository nor checking for upgrades is not supported.
+You need to provide full path to MSI pakage file to install, or installed
+package name (and optionaly version) to uninstall.
+
+**Example**: install [Google Chrome][] but prevent it from self-upgrading
+(otherwise Google Chrome's self-upgrading will conflict with CFEngine ensuring
+that version from this particluar MSI is installed):
+
+[Google Chrome]: https://cloud.google.com/chrome-enterprise/browser/download/#chrome-browser-update
+
+```cf3
+  packages:
+    windows::
+      "C:\GoogleChromeStandaloneEnterprise.msi"
+        policy => "present",
+        package_module => msiexec;
+
+      "Google Update Helper"
+        policy => "absent",
+        package_module => msiexec;
+```
+
+**History:**
+
+* Added in CFEngine 3.14.0
```

Comment:
We are backporting this to 3.12.x aren't we? This can be updated when we do it.

Reply:
Right. But we can already add this info